react/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { canUseDOM } from 'vtex.render-runtime'
+import { PixelMessage } from './typings/events'
+import { fetchEmail, getSiteType } from './helpers'
+
+function impression(event: PixelMessage) {
+  switch (event.data.eventName) {
+    case 'vtex:pageInfo':
```

Comment:
Pageinfo is triggered in other pages, not just in home page

Reply:
Added the raw `eventType` as a fallback just in case..